### PR TITLE
Set min and max controller versions for 1.8.1

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

File: buildsettings.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "version": "1.8.1",
     "avi": {
-        "maxVersion": "21.1.3",
-        "minVersion": "20.1.5"
+        "maxVersion": "22.1.2",
```
Comment (apalsule) [inline buildsettings.json:4]: Can you make this 22.1.1

**Comment by akshayhavile** [comment]
Doc PR. setting up appropriate Version. UT failures are not related to it.

**Pull request merged**